openage/convert/processor/export/generate_hash.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020-2020 the openage authors. See copying.md for legal info.
+"""
+Provides functions for traversing a directory and
+generating hash values for all the items inside.
+"""
+
+import hashlib
+import os
+
+
+HASHING_FUNCTION = 'sha3_256'
+BUF_SIZE = 32768
```

Comment:
For less hardcoding, these should be moved into `generate_hashes(..)` as optional parameters. Then we can change the hashing function and buffer size dynamically if we need to.

I'm thinking of something along the lines of 

```
generate_hashes(modpack, exportdir, hashfunc='sha3_256', bufsize=32768):
```